app/src/main/java/live/videosdk/rtc/android/java/ParticipantAdapter.java

```diff
@@ -132,6 +132,13 @@ public void onStreamDisabled(Stream stream) {
         holder.btnMenu.setOnClickListener(v -> showPopup(holder, finalParticipant));
     }
 
+    @Override
+    public void onViewDetachedFromWindow(@NonNull PeerViewHolder holder) {
+        holder.svrParticipant.release();
+        holder.svrParticipant.clearImage();
+        super.onViewDetachedFromWindow(holder);
+    }
+
```

Comment:
Causes blank video when scrolled out of view and back in

![image](https://user-images.githubusercontent.com/17140551/155379331-821efa8d-c7cc-47f3-9b4e-f1e851de8082.png)